MaxText/maxengine.py

```diff
@@ -115,7 +111,7 @@ def model_apply(_p, _rng):
       params = {}
       params['aqt'] = new_vars['aqt']
       # Remove param values which have corresponding qtensors in aqt to save memory.
-      params['params'] = quantizations.remove_quantized_params(state.params, new_vars['aqt'])
+      params['params'] = quantizations.remove_quantized_params(state.params['params'], new_vars['aqt'])
```

Comment:
Have to do state.params['params'] here because we are breaking away from the flax world that deals with {"params: stuff} nicely and doing our own thing in quantizations. _sigh_